package.json

```diff
@@ -33,18 +33,17 @@
   "scripts": {
     "//eslint": "performs static analysis over project files and examples",
     "eslint": "eslint --fix lib/*.ts test/*.js examples/**/*.js",
-
     "//prettier": "performs code formatting over project files and examples",
     "prettier": "prettier --write lib/*.ts test/*.js examples/**/*.js",
-
     "//prepare": "prepare project for distribution",
     "prepare": "tsc",
-
     "//test": "runs tests on main project",
```

Comment:
I think the blank lines were removed by the npm tool when I ran `npm -i -D ....`